classes/kernel-cve-check.bbclass

```diff
@@ -28,9 +28,14 @@ KERNEL_CVE_CHECK_SHOW_IGNORE_INFO ?= "0"
 KERNEL_CVE_CHECK_LINUX_GIT_REMOTE_REPO ??= ""
 KERNEL_CVE_CHECK_LINUX_GIT_REMOTE_AUTODETECT ??= "1"
 
+KERNEL_PN = ""
+KERNEL_CVE_CHECK_TASK_ORDER = ""
+
 python() {
-    if d.getVar("PN") == "linux-base":
+    if d.getVar("PN") in ['linux-base', 'linux-k510']:
```

Comment:
In the future, we could make this more generic way instead of using recipe names.

Reply:
We could use the condition that CVE_PRODUCT is set to "linux_kernel".